aws-ec2-manager-blueprint-inputs.yaml

```diff
@@ -62,12 +62,20 @@ aws_secret_access_key: ''
 # NOTE: If enabled, the certificate and private key files must reside in resources/ssl.
 #ssl_enabled: false
 
+# If SSL is enabled, certificates are verified by default.
+# To disable certificate verification, set this value to false.
+#verify_certificate: false
```

Comment:
after further check, this input is used for both all requests to the rest manager, not just for agent hosts, so I think the name is ok